.github/workflows/ci.yaml

```diff
@@ -25,19 +23,21 @@ jobs:
         echo "##[set-env name=GOPATH;]$(go env GOPATH)"
         echo "##[add-path]$(go env GOPATH)/bin"
     - name: Install ko
-      run: go get github.com/google/ko/cmd/ko
+      run: ( cd .. && GO111MODULE=on go get github.com/google/ko/cmd/ko )
```

Comment:
Seems a bit dangerous to use `..`, wouldn't it make more sense to use a temp dir.
As a matter of fact, given there is no version defined here (nor in a go.mod file that isn't in ..), why not force GO111MODULE=_off_?

Reply:
why is `..` dangerous? The directory is guaranteed to exist. It is guaranteed not be in a go module. It won't be mutated

The point of moving out of the current dir is to insure that the go.mod of the provisioner will never conflict with `ko`, or be mutated when installing `ko`. We turn GO111MODULE on because ko defines it's dependencies as a go module and should be treated as such. We don't use a version because ko isn't versioned. In short, `go get` is a tire fire, but it's what we have.

This is a pattern we use widely across projectriff.

Reply:
Won't that create a go.mod file in `..` ?

Reply:
> Won't that create a go.mod file in .. ?

It does not